tutorials/optimization/optimizing_3d_performance.rst

```diff
@@ -22,57 +22,64 @@ in the street you are in, as well as the sky and a few birds flying overhead. As
 far as a naive renderer is concerned however, you can still see the entire town.
 It won't just render the buildings in front of you, it will render the street
 behind that, with the people on that street, the buildings behind that. You
-quickly end up in situations where you are attempting to render 10x, or 100x
-more than what is visible.
+quickly end up in situations where you are attempting to render 10× or 100× more
+than what is visible.
 
```

Comment:
This paragraph refers to the paragraph that used to be directly above and should be moved back up. The added paragraph above can go under this one just fine as it is application to more than just PVS